default.json

```
@@ -1,7 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "timezone": "Asia/Tokyo",
+  "schedule": ["after 1am and before 3am on the first day of the month"],
```

```suggestion
  "schedule": ["before 3am on the first day of the month"],
```

**Suggested**: Presetにもありますのでこちら使ってもいいと思います。
https://docs.renovatebot.com/presets-schedule/#schedulemonthly